firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionRegistrar.java

```diff
@@ -41,7 +41,9 @@
             .factory(
                 c ->
                     new FirebaseAppDistribution(
-                        c.get(FirebaseApp.class), c.get(FirebaseInstallationsApi.class)))
+                        c.get(FirebaseApp.class),
+                        c.get(FirebaseInstallationsApi.class),
+                        new FirebaseAppDistributionTesterApiClient()))
```

Comment:
I have a feeling there is another way to do this without calling new. Can we add a TODO to look into FirebaseComponents and make sure this is an okay way to initialize the TesterApiClient?